addons/point_of_sale/models/pos_config.py

```diff
@@ -545,6 +552,18 @@ def open_opened_session_list(self):
             'domain': [('state', '!=', 'closed'), ('config_id', '=', self.id)]
         }
 
+    def open_opened_rescue_session_list(self):
+        self.ensure_one()
+
+        return {
+            'name': _('Opened Sessions'),
+            'res_model': 'pos.session',
+            'view_mode': 'form',
+            'res_id': self.session_ids.filtered(lambda s: not s.state == 'closed' and s.rescue).id,
+            'type': 'ir.actions.act_window',
+            'domain': [('state', '!=', 'closed'), ('config_id', '=', self.id), ('rescue', '=', True)]
```

Comment:
No need for a domain if you are already specifying which session you want to display in `res_id`